ksqldb-engine/src/main/java/io/confluent/ksql/function/udf/math/Least.java

```diff
@@ -43,7 +43,7 @@
   @Udf
   public Integer least(@UdfParameter final Integer val, @UdfParameter final Integer... vals) {
 
-    return Stream.concat(Stream.of(val), Arrays.stream(vals))
+    return (vals == null) ? null : Stream.concat(Stream.of(val), Arrays.stream(vals))
```

Comment:
Here and below, similar to JsonConcat and Greatest, can `vals` actually be null?